lightning/src/ln/channelmanager.rs

```diff
@@ -3522,6 +3529,60 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		}
 	}
 
+	/// Gets the set of txids which should be monitored for their confirmation state.
+	///
+	/// If you're providing information about reorganizations via [`transaction_unconfirmed`], this
+	/// is the set of transactions which you may need to call [`transaction_unconfirmed`] for.
+	///
+	/// This may be useful to poll to determine the set of transactions which must be registered
+	/// with an Electrum server or for which an electrum server needs to be polled to determine
+	/// transaction confirmation state.
+	///
+	/// This may update after any [`transactions_confirmed`] or [`block_connected`] call.
+	///
+	/// Note that this is NOT the set of transactions which must be included in calls to
+	/// [`transactions_confirmed`] if they are confirmed, but a small subset of it.
```

Comment:
I assume the rest will be registered in one of the `register_tx` calls? Could be worth adding a note, since this line may come across naively as a bit scary but unactionable...

Nw though if Jeff is addressing the docs more thoroughly in #858.

Reply:
I believe the plan is still for @jkczyz to do a followup PR after this + #858 that will add a new top-level trait similar to `chain::Listen` and merge all the docs/functions then.